demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -239,17 +239,46 @@ def convert_contribution_dir_to_pack_contents(self, unpacked_contribution_dir: s
             dst_name = ENTITY_TYPE_TO_DIR.get(basename, '')
             src_path = os.path.join(self.pack_dir_path, basename)
             dst_path = os.path.join(self.pack_dir_path, dst_name)
-            if os.path.exists(dst_path):
-                # move src folder files to dst folder
-                for _, _, files in os.walk(src_path, topdown=False):
-                    for name in files:
-                        src_file_path = os.path.join(src_path, name)
-                        dst_file_path = os.path.join(dst_path, name)
-                        shutil.move(src_file_path, dst_file_path)
-                shutil.rmtree(src_path, ignore_errors=True)
-            else:
-                # replace dst folder with src folder
-                shutil.move(src_path, dst_path)
+            # move src folder files to dst folder
+            for _, _, files in os.walk(src_path, topdown=False):
+                for name in files:
+                    src_file_path = os.path.join(src_path, name)
+                    dst_file_path = self.fix_dst_path(basename, name, dst_path, src_file_path)
+                    shutil.move(src_file_path, dst_file_path)
+            shutil.rmtree(src_path, ignore_errors=True)
+
+    def fix_dst_path(self, basename, file_name, dst_dir_path, src_file_path):
+        new_dst_file_path = None
+        if basename == "classifier":
+            new_dst_file_path = self.fix_dst_file_path_for_mapper(file_name, dst_dir_path, src_file_path)
+        elif basename == "incidentfield":
+            new_dst_file_path = self.fix_dst_folder_path_for_fields(file_name)
+        if not new_dst_file_path:
```

Comment:
Unless we have a good reason
```suggestion
        else:
```

Reply:
In case the base name is classifier and the file is really a classifier we will get None from `fix_dst_file_path_for_mapper` func, so we need this condition to make sure if it is None or not.